lms/services/email_preferences.py

```diff
@@ -78,11 +113,10 @@ def _url(self, route, h_userid, tag):
             _query={"token": self._encode_token(TokenPayload(h_userid, tag))},
         )
 
-    def unsubscribe(self, h_userid):
+    def instructor_digest_unsubscribe(self, h_userid) -> None:
```

Comment:
Longer function name, includes the type of email.

I don't think we want a "system" to unsubscribe from arbitrary emails as they might (and will) have different storage characteristics.